tests/test_get_vin_history.py

```diff
@@ -81,3 +81,22 @@ def test_vin_non_validate(self):
         vin = '2FMGK5D81EBD1433'
         response = client.get_vin_history(vin)
         self.assertEqual(response, 'Vin is not valid: {}'.format(vin))
+        
+        
+    def test_get_vin_history_with_pagination(self):
+        
+        token = os.getenv('API_TOKEN')
+        client = MarketAPI(token)
+
+        vins = {"1FTEW1EF1FFA67753":6,}
+
+        for vin,page_limit in vins.items():
+            for count in range(1,page_limit+1):
+                time.sleep(3)
```

Comment:
Is the `time.sleep` added here for ratelimit ?
If it is,  once you tested and it works with the ACTUAL api token,  please mock out the response and remove the `time.sleep` here